docs/index.asciidoc

```diff
@@ -8,9 +8,9 @@
 :plugins-repo-dir:      {docdir}/../../logstash-docs/docs
 :branch:                6.x
```

Comment:
@jsvd. Based on past releases, I think `branch` should be set to `6.5`.   Currently, when you click a link in the published 6.5 doc, it takes you to the 6.x version. 

I didn't see this in the release checklist, but I'll be happy to add it.